lib/tbot/internal/heartbeat/service.go

```diff
@@ -182,6 +200,56 @@ func (s *Service) heartbeat(ctx context.Context, isOneShot, isStartup bool) erro
 		return trace.Wrap(err, "submitting heartbeat")
 	}
 
-	s.cfg.Logger.InfoContext(ctx, "Sent heartbeat", "data", hb.String())
+	s.cfg.Logger.DebugContext(ctx, "Sent heartbeat", "data", hb.String())
 	return nil
 }
+
+func (s *Service) watchStatusChanges() (<-chan struct{}, func()) {
+	notifCh, unsubscribe := s.cfg.StatusRegistry.Watch()
+
+	reloadCh := make(chan struct{})
+	closeCh := make(chan struct{})
+
+	go func() {
+		for {
+			select {
+			case <-notifCh:
+			case <-closeCh:
+				return
+			}
```

Comment:
I'm thinking we'll extend this `select` to send heartbeats when notices are emitted.